create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/check-requirements.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import os
+import click
+from functools import partial
+from pathlib import Path
+
+if __name__ == '__main__':
+    path = Path('../../requirements/')
+
+    for file in os.listdir(path):
```

Comment:
U use `Path` from `pathlib`, so u can use built-in method for it 😀

*simple example*

```python
import pathlib

requirements_dir = pathlib.Path('./requirements')


def update_requirements() -> None:
    req_files = [req_file for req_file in requirements_dir.iterdir() if req_file.is_file()]

    with req_file.open() as f:
        # do your stuff
```

[link to docs](https://docs.python.org/3/library/pathlib.html#basic-use)